src/components/projects-summary/projects-summary.html

```diff
@@ -55,10 +49,13 @@ <h3 class="project-tile-header">
           <a href="" ng-click="$ctrl.handleProjectClick(project)" class="project-title">{{project | displayName}}</a>
         </h3>
         <p class="project-date">
+          <span ng-if="project | displayName : true"><span ng-bind-html="project.metadata.name | highlightKeywords : keywords"></span> &ndash;</span>
+          created
+          <span ng-if="project | annotation : 'openshift.io/requester'">by <span ng-bind-html="project | annotation : 'openshift.io/requester' | highlightKeywords : keywords"></span></span>
           {{project.metadata.creationTimestamp | date: 'mediumDate'}}
```

Comment:
@spadgett I agree with you on the oddity of only making the change in one place. Customers have asked to see explicit dates rather than relative dates, but we can wait to make that change (post MVP) once we can change the full projects list as well. @jeff-phillips-18 Does that work for you?